kernel/sysctl_setter.go

```diff
@@ -85,7 +85,7 @@ func (s *SysctlSetterImpl) Exists() bool {
 	}
 
 	if _, err := Parameter(s.paramName); err != nil {
-		return strings.Contains(err.Error(), "cannot stat")
+		return !strings.Contains(err.Error(), "cannot stat")
```

Comment:
Why any other than "cannot stat" error is OK ? Maybe all/any error means parameter does not exist in a form we can use it?